README.md

```diff
@@ -21,4 +21,44 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ### Quick Start
 
-Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
\ No newline at end of file
+Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
+
+### Where to get the vSphere provider container
+
+The containers for this provider are currently hosted at `gcr.io/cnx-cluster-api/`.  Each release of the
+container are tagged with the release version appropriately.
+
+| release version | container url |
+| --- | --- |
+| 0.1 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:v0.1 |
+| 0.2 | gcr.io/cnx-cluster-api/vsphere-cluster-api-provider:0.2 |
+
+To use the appropriate version (instead of `:latest`), replace the version in the generated `provider-components.yaml`
+described in the quick start guide.
+
+### Compatibility Matrix
+
+Below are tables showing the compatibility between versions of the vSphere provider, the main cluster api,
+kubernetes versions, and OSes.  Please note, this table only shows version 0.2 of the vSphere provider.  Due
```

Comment:
what if we pull `v0.1` altogether to remove confusion?

Reply:
The problem with v0.1 was caused by the Ubuntu package management.  It also caused a problem with v0.2 but we caught it in time and fixed it before the release.  There is still a chance that the release process for Ubuntu and k8s will again break v0.2.  There is also a chance that it will fix v0.1.

I think until we move away from the OS package management (which is an unknown variable we do not control), we can never remove uncertainty.  I'm in favor of keeping v0.1 because of this.

Reply:
👍

Reply:
btw, when we did a release last time, we just pushed an `official` container but did not do a github release.  I'm preparing this for v0.2, and it does look kind of weird having no v0.1 in the github release section.